cmd/gemini/root.go

```diff
@@ -94,8 +94,8 @@ var (
 	testClusterHostSelectionPolicy   string
 	oracleClusterHostSelectionPolicy string
 	useServerSideTimestamps          bool
-	requestTimeout					 time.Duration
-	connectTimeout					 time.Duration
+	requestTimeout                   time.Duration
```

Comment:
Please, move it out the commit.
I am ok to see it in the same PR, but separate commit.

The `git commit -p` will allow to do it easily.

Reply:
done